packages/agent/tests/sync-manager.spec.ts

```diff
@@ -1,16 +1,20 @@
-import type { PortableDid } from '@web5/dids';
 
-import { expect } from 'chai';
-import * as sinon from 'sinon';
-import { RecordsQueryReply, RecordsWriteMessage } from '@tbd54566975/dwn-sdk-js';
+import sinon from 'sinon';
+import chai, { expect } from 'chai';
+import chaiAsPromised from 'chai-as-promised';
 
 import type { ManagedIdentity } from '../src/identity-manager.js';
+import type { PortableDid } from '@web5/dids';
+
+import { RecordsQueryReply, RecordsQueryReplyEntry, RecordsWriteMessage } from '@tbd54566975/dwn-sdk-js';
```

Comment:
Minor nit.  Web5 JS imports are sorted:
- external package type imports
- external package imports
- internal type imports
- internal imports

```suggestion
import type { PortableDid } from '@web5/dids';
import type { RecordsQueryReply, RecordsQueryReplyEntry, RecordsWriteMessage } from '@tbd54566975/dwn-sdk-js';

import sinon from 'sinon';
import chai, { expect } from 'chai';
import chaiAsPromised from 'chai-as-promised';

import type { ManagedIdentity } from '../src/identity-manager.js';
```